### PR TITLE
Update archetype IntegrationTestMojo to use InputCombinations

### PR DESCRIPTION
**Pull request by batsatt**

### CODE REVIEW

File: maven-plugins/helidon-archetype-maven-plugin/test.log [hunk 1]
```diff
@@ -0,0 +1,1272 @@
+[INFO] Scanning for projects...
```
Comment (romain-grecourt) [inline maven-plugins/helidon-archetype-maven-plugin/test.log]: Is this file intended ?

Comment (batsatt) [reply maven-plugins/helidon-archetype-maven-plugin/test.log]: whoops, no, I'll remove it.

**Review by romain-grecourt — APPROVED** [review]

**Comment by batsatt** [comment]
Fixes #589
